minique/_compat.py

```diff
@@ -1,4 +1,4 @@
 try:
     from typing import TYPE_CHECKING  # "New in version 3.5.2."
-except ImportError:
+except ImportError:  # pragma: no cover
```

Comment:
What does this comment mean?

Reply:
It marks the line (or in the case of block-starting lines such as this, the whole block) ignored by coverage checking.

That is, we don't care whether or not our tests cover this bit – in this case, because it's not worth it to test on 3.5.1 or 3.5.0 just for this.